src/resources/js/views/events-bar-inputs.js

```diff
@@ -97,7 +97,7 @@ tribe.events.views.eventsBarInputs = {};
 		$container
 			.find( obj.selectors.inputWrapper )
 			.each( function( index, wrapper ) {
-				var inputWrapperClass = wrapper.className.match( /tribe-common-c-search__input-control--[a-z]+/ );
+				var inputWrapperClass = wrapper.className.match( /tribe-events-c-search__input-control--[a-z]+/ );
```

Comment:
Are these fixing bugs? Should this be ticketed somewhere and done separately from the doc PR?

Reply:
Can do.